generic3g/specs/FrequencyAspect.F90

```diff
@@ -0,0 +1,140 @@
+#include "MAPL_Generic.h"
+module mapl3g_FrequencyAspect
+   use mapl3g_StateItemAspect
+   use mapl3g_AccumulatorActionInterface
+   use esmf
+   implicit none
+   private
+
+   public :: FrequencyAspect
+
+   type, extends(StateItemAspect) :: FrequencyAspect
+      type(ESMF_TimeInterval) :: dt
```

Comment:
Because of the delayed evaluation of dt, you will almost certainly need a `set_dt()` method here.

Reply:
And ... I'm starting to really prefer timestep.   Let's discuss again when we can.

Reply:
I added setters for dt and accumulation.